File: video/v1/room/create-group-small-room/create-group-small-room.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$room = $twilio->video->v1->rooms
                          ->create(array(
                                       "recordParticipantsOnConnect" => True,
                                       "statusCallback" => "http://example.org",
                                       "type" => "group-small",
                                       "uniqueName" => "SmallDailyStandup"
                                   )
                          );

print($room->sid);

File: video/v1/room/create-group-small-room/create-group-small-room.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

room = @client.video.rooms.create(
                             record_participants_on_connect: true,
                             status_callback: 'http://example.org',
                             type: 'group-small',
                             unique_name: 'SmallDailyStandup'
                           )

puts room.sid
